schemas/common_definitions.json

```diff
@@ -191,9 +191,9 @@
         "goto": {
           "type": "object",
           "properties": {
-            "id": {
+            "block": {
               "type": "string",
-              "description": "The id of another block in the same group."
+              "description": "The block in the same group."
```

Comment:
This description isn't very good. you should describe better what this value is and how it is used.